GitHubAction/UploadArtifactApi/HttpArtifactUploadApi.cs

```diff
@@ -43,10 +49,10 @@ public HttpArtifactUploadApi(HttpClient httpClient)
 
         if (response.StatusCode is HttpStatusCode.Forbidden or HttpStatusCode.Unauthorized)
         {
-            throw new KeyException($"The upload api returned a {response.StatusCode} response.");
+            throw new KeyException($"The upload api returned a {response.StatusCode} response. Body:" + response?.Content?.ToString() ?? "null");
```

Comment:
```suggestion
            throw new KeyException($"The upload api returned a {response.StatusCode} response. Body: {response.Content}");
```

Reply:
Content will never be null
![image](https://github.com/SkylineCommunications/Skyline-DataMiner-Deploy-Action/assets/97458010/88699f9b-1c5d-463e-95fa-41ca1d8ac7ce)